llvm/lib/Target/NVPTX/NVPTXISelLowering.cpp

```diff
@@ -5557,6 +5557,50 @@ static SDValue PerformLOADCombine(SDNode *N,
       DL);
 }
 
+// Lower a v16i8 (or a v8i8) store into a StoreV4 operation with i32 results
+// instead of letting ReplaceLoadVector split it into smaller stores during
+// legalization. This is done at dag-combine1 time, so that vector operations
```

Comment:
```suggestion
// legalization. This is done at dag-combine time, so that vector operations
```

Reply:
Are you sure about this? This is intended to be done before legalization. Changed to dag-combine.

Reply:
My mistake; I read it as a typo